tests/PHPUnit/bootstrap.php

```diff
@@ -88,7 +88,7 @@ function prepareServerVariables()
 
 // General requirement checks & help: a webserver must be running for tests to work if not running UnitTests!
 if (empty($_SERVER['argv']) || !in_array('UnitTests', $_SERVER['argv'])) {
-    checkPiwikSetupForTests();
+//    checkPiwikSetupForTests();
```

Comment:
little error here - btw maybe we have a problem with this function if you need to comment it?